src/components/common/filtering/ProjectsListItem/ProjectListItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { ProjectLite } from "../project";
+import {
+  TitleSection,
+  Wrapper,
+  Title,
+  CommitmentSection,
+  Subtitle,
+  Paragraph,
+  PositionsList,
+  PositionsListItem,
+  Button,
+  PositionLevel,
+  PositionTitle,
+  TypeSection,
+  PositionsSection,
+} from "./StyleProjectListItem";
+
+interface Props {
+  project: ProjectLite;
+}
+
+export default function ProjectListItem({ project }: Props) {
+  return (
+    <Wrapper>
+      <TitleSection>
+        <Title>{project.title}</Title>
+        <Paragraph>{project.catchPhrase}</Paragraph>
+      </TitleSection>
+      <TypeSection>
+        <Subtitle>Type</Subtitle>
+        <Paragraph>Idea - Platform</Paragraph>
+      </TypeSection>
+      <PositionsSection>
+        <div>
+          <Subtitle>Positions Available / Level</Subtitle>
+          {project.openPositions.length > 0 ? (
+            <>
+              <PositionsList>
+                {project.openPositions.slice(0, 3).map((position) => (
+                  <PositionsListItem key={position.id}>
+                    <PositionTitle>{position.title}</PositionTitle>
+                    <PositionLevel>Advanced</PositionLevel>
+                  </PositionsListItem>
+                ))}
+              </PositionsList>
+            </>
+          ) : (
+            <Paragraph>No open positions for the moment</Paragraph>
```

Comment:
since this is a portal for open positions, we don't really want to have a card with no open positions at all.